app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -120,20 +123,27 @@ const rateLimitTimeouts = {};
  *  MetaMetricsController
  * @param {Function} opts.getMetricsState - get the state of
  *  MetaMetricsController
- * @param {number} [opts.rateLimitSeconds] - number of seconds to wait before
- *  allowing another set of events to be tracked.
+ * @param {number} [opts.rateLimitTimeout] - time, in milliseconds, to wait before
+ *  allowing another set of events to be tracked for methods rate limited by timeout.
+ * @param {number} [opts.rateLimitSamplePercent] - percentage, in decimal, of events
+ *  that should be tracked for methods rate limited by random sample.
  * @param {Function} opts.getAccountType
  * @param {Function} opts.getDeviceModel
  * @param {RestrictedControllerMessenger} opts.snapAndHardwareMessenger
  * @param {AppStateController} opts.appStateController
+ * @param opts.globalRateLimitTimeout
```

Comment:
Nit: A JSDoc type and description would be great here

Reply:
https://github.com/MetaMask/metamask-extension/pull/23257/commits/6c967fa3e185677be91b57d98360339c9f3b43aa